index.bs

```diff
@@ -16,7 +16,7 @@ Version History: https://github.com/w3c/gyroscope/commits/gh-pages/index.bs
 Indent: 2
 Repository: w3c/gyroscope
 Markup Shorthands: markdown on
-Inline Github Issues: true
+Inline Github Issues: title
 !Test Suite: <a href="https://github.com/w3c/web-platform-tests/tree/master/gyroscope">web-platform-tests on GitHub</a>
 Boilerplate: omit issues-index, omit conformance
 Default Biblio Status: current
```

Comment:
@pozdnyakov https://github.com/w3c/sensors/pull/157 landed so you may want to sync accordingly.